app/styles/components/c-cover-page.scss

```diff
@@ -189,6 +193,25 @@
         transform: rotate(45deg);
         width: 14px;
       }
+
+      &::after {
+        border-bottom: 1.5px solid $color-4;
+        content: '';
+        height: 5px;
+        margin-left: -15px;
+        margin-top: 10px;
+        position: absolute;
+        top: 0;
+        width: 15px;
+      }
+
+      &:last-child {
+
+        &::after {
+          content: '';
```

Comment:
You don't need to specify the two properties: by just setting `content` to `none`, the pseudo element will disappear.